### ISSUE TITLE
Error on blog management content / data screen

### ISSUE THREAD

**Issue by mcalado** [2sic/app-blog#4]
We have the following error on blog management content / data screen when we try to go to this page to delete a blog. Do you know where we could look at it to fix? I'm trying to check the database data, any tip is welcome, thank you.

Message:An item with the same key has already been added.

StackTrace:

   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at ToSic.Eav.Serializers.Serializer.GetDictionaryFromEntity(IEntity entity, String language) in C:\Projects\eav-server\ToSic.Eav.WebApi\Serializers\Serializer.cs:line 158
   at ToSic.SexyContent.Serializers.Serializer.GetDictionaryFromEntity(IEntity entity, String language) in C:\Projects\2SexyContent\Web\DesktopModules\ToSIC_SexyContent\SexyContent\Serializers\Serializer.cs:line 59
   at ToSic.Eav.Serializers.Serializer.b__25_0(IEntity c) in C:\Projects\eav-server\ToSic.Eav.WebApi\Serializers\Serializer.cs:line 100
   at System.Linq.Enumerable.<>c__DisplayClass12`3.b__11(TSource x)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at ToSic.Eav.WebApi.EntitiesController.GetAllOfTypeForAdmin(Int32 appId, String contentType) in C:\Projects\eav-server\ToSic.Eav.WebApi\EntitiesController.cs:line 67
   at lambda_method(Closure , Object , Object[] )
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass13.b__c(Object instance, Object[] methodParameters)
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.<>c__DisplayClass5.b__4()
   at System.Threading.Tasks.TaskHelpers.RunSynchronously[TResult](Func`1 func, CancellationToken cancellationToken)

**Comment by mcalado** [comment]
![blogerror](https://cloud.githubusercontent.com/assets/34480/19525973/6c73d606-95e8-11e6-96b7-c82df081d56f.PNG)

**Comment by iJungleboy** [comment]
Just a guess - maybe a caching issue - which would need a dnn-restart (touch the web.config). 

Another guess - based on the code line number
https://github.com/2sic/eav-server/blob/master/ToSic.Eav.WebApi/Serializers/Serializer.cs#L158

This one seems to find a problem with a draft. I'm not 100% sure, but maybe you did something, which caused an item to think that it is a) published and b) the draft of itself. This shouldn't happen - and I can't guess how this would - but maybe it did? If this is the case, please tell me how you got there (that would be something the software should really capture), and just fix it in the sql table.

**Comment by mcalado** [comment]
Thank you for the feedback. The caching approach didn't resolve it. We are able to get it resolved, here is what happens, seems to be a specific scenario: we updated the blog template with customization, we changed the published (eye icon) to false for the default blog post that came with the module, when we change it back the error above is resolved. We tried to delete this post but it warning saying that it's used by a template, when we select the force delete option, it crashed all the other blog posts built. We restored the environment and we just keep the original blog post as unpublished and updated it's name to remind us that it shouldn't be deleted. Thanks.
